tests/integration/logs.spec.ts

```diff
@@ -53,6 +53,18 @@ describe('Logs', function () {
 				},
 			]);
 			await delay(LOG_PROPAGATION_TIMEOUT);
+
+			// Wait for both logs to propagate
+			await waitFor(
```

Comment:
If we really need this, then let's move it inside the `should successfully load historical logs` test (replacing what it atm has)